### PR TITLE
Fix send embed to webhook action

### PR DESCRIPTION
**Pull request by ForceBlood**
**Please describe the changes this PR makes and why it should be merged:**

**Status**

- [x] Code changes have been tested against the Discord API and the discord.js wrapper, or there are no code changes
- [x] Documentation has been added/modified, or there is nothing to change (docs/mods.json)

**Semantic versioning classification:**

- [ ] This PR changes DBM's interface (methods or parameters added to default methods)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Comment by ForceBlood** [comment]
Fixes #886

**Comment by OneAndonlyFinbar** [comment]
`<WebhookClient>#send` Returns a promise

File: actions/send_embed_to_webhook_MOD.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ module.exports = {
 
     if (!embed2 || !webhook) return this.callNextAction(cache);
 
-    webhook.send(embed2);
+    await webhook.send({ embeds: [embed2] });
     this.callNextAction(cache);
```
Comment (almeidx) [inline actions/send_embed_to_webhook_MOD.js:66]:
```suggestion
    webhook.send({ embeds: [embed2] })
      .then(() => this.callNextAction(cache))
      .catch((err) => this.displayError(data, cache, err));
```

**Review by almeidx — CHANGES_REQUESTED** [review]
https://github.com/dbm-network/mods/pull/900#discussion_r945828063

**Review by almeidx — APPROVED** [review]

**Pull request merged**